docs/docs/HowTo/schemas/Approve-a-schema.md

```diff
@@ -0,0 +1,40 @@
+# Manage schema
+
```

Comment:
Also this file is called Approve-a-schema but should probably be renamed as manage-schemas

I created this file originally so my bad :/